webapp/src/Controller/API/ProblemController.php

```diff
@@ -258,6 +258,137 @@ public function addProblemAction(Request $request): array
         ];
     }
 
+    /**
+     * Unlink a problem from this contest.
+     * @Rest\Delete("/{id}")
+     * @IsGranted("ROLE_ADMIN")
+     * @OA\Response(response="204", description="Deleting problem succeeded")
+     * @OA\Parameter(ref="#/components/parameters/id")
+     */
+    public function deleteProblemAction(Request $request, string $id): Response
```

Comment:
Maybe the contest context is not obvious to everyone. So it might be helpful to name it `deleteContestProblem`?

Reply:
But should it then also be listInContestAction, singleInContestAction, etc? With that I mean: we have many more actions without problem in the name.

Reply:
Perhaps we should? I would personally also get nervous from a message like "Deleting problem succeeded" while it was only unlinked. Maybe that's just me

Reply:
Of course that message is not what you get back, thats the description of the API doc, where it already says "Unlink a problem from this contest.":
<img width="801" alt="image" src="https://user-images.githubusercontent.com/550145/160121398-42682a44-0a0a-4daa-a7ff-3dbf87e3af29.png">

I do see we say 'the ID of the entity to get' there btw, which is not right.